### ISSUE TITLE
vue addon-docs display prop values in prop table.

### ISSUE THREAD

**Issue by travis-w** [storybookjs/storybook#9851]
**Is your feature request related to a problem? Please describe.**
I'm not sure if this is already possible, but I have looked around and tried a bunch of different options, but I would like the possible values for a component prop to show up in the prop table in docs.

_This may already be possible, but it seems that all of my jsdoc comments are being pulled except for the @values field. Please point me in the right direction if so._

**Describe the solution you'd like**
Grab @values from jsdoc on component and display in addon-docs prop table. Displaying under the description would be nice, or another column all together.

**Describe alternatives you've considered**
It looks like this may be possible by describing the props with PropTypes in your story, but since I am using, Vue, that would be an extra dependency that only gets used for this feature. JSDoc has @values so it would be nice to have that pulled in as well.

**Are you able to assist bring the feature to reality?**
Possibly. If someone could point me in the right direction

**Additional context**
Here is the image that shows what I would like:
![Example](https://raw.githubusercontent.com/storybookjs/storybook/master/addons/docs/docs/media/docspage-hero.png)
It shows the possible values in the description. This is from the docs documentation so it looks like it may be possible, but just not working for me.

**Comment by shilman** [comment]
Hi @travis-w! I'm currently setting up a proper test framework for props table in  https://github.com/storybookjs/storybook/pull/9838 and that should be ready this week. Once that's in place, it should be pretty straightforward to add this and I'd love to work with you on that. I'll post back here once it's merged, and if you want to pair on it, feel free to hop on our discord https://discord.gg/UUt2PJb

**Comment by travis-w** [comment]
@shilman That sounds great! I've joined the discord and look forward to working on this.

**Comment by shilman** [comment]
@travis-w Awesome!! I just went through all the 40-something open props issues and created repros for most of them. The new setup makes it pretty efficient. Unfortunately, Vue is the only major framework I haven't gotten working yet. 

Attempted test setup here: https://github.com/storybookjs/storybook/pull/9841

Hoping @Aaron-Pool can get me unblocked here! 🙏🙏🙏

**Comment by mathijsblokland** [comment]
I've also been looking for this as i'm setting up storybook for our Vue app and wanted to show the options (values) for a specific property. In my case i'd like to have the values defined in JSDoc to be populated as options into a controls dropdown. This would make it easier to switch between classes on an element, but listing those values in a separate column would also be of great help 😄.

**Comment by fallemand** [comment]
Has it been any update on this? Could be super useful.
This is also not working in vue:
```vue
size: {
  default: 'medium',
  type: String as PropType<'button' | 'submit' | 'reset'>,
},
```
![image](https://user-images.githubusercontent.com/16105726/100556715-fd324700-3282-11eb-8f77-38f1a43862ae.png)
